### ISSUE TITLE
Is there any big difference between this and generator?

### ISSUE THREAD

**Issue by Melindrea** [assemble/assemble-example#2]
As in, if you follow the installation guide on this project, will you get the same thing as if you did `yo assemble`? If not, what are the differences, and is there any particular reason for it?

To my view of things, it's good to have a couple of different ways to do things, but it should also give about the same result.

**Comment by jonschlinkert** [comment]
Good question! I may have used this as the boilerplate in the generator, but I'll check to be sure. 

I agree with your point as well, we should try to make sure that the boilerplates/generators provide a consistent experience.

**Comment by Melindrea** [comment]
Which shouldn't be too difficult, should it? Just need to deploy a generated one to this one every time it changes.

At least in the generator I'd think it could be a good idea to allow for a verb-based documentation. I hadn't spotted verb until I started to go through the repos on here, and now I'm trying to figure out how to quickest implement it =)

**Comment by jonschlinkert** [comment]
> Which shouldn't be too difficult, should it? 

Not at all. I can actually use the boilerplate in the generator. By way of example, take a look at what I'm doing with generator-verb: https://github.com/assemble/generator-verb/blob/master/doc/index.js#L13 and https://github.com/assemble/generator-verb/blob/master/doc/index.js#L35

Essentially, when you run, say, `yo verb:doc usage`, generator-verb will copy the `usage.md` template [from here](https://github.com/assemble/verb-readme-includes/tree/master/templates) to the `./docs` directory in your project. 

We can do the same for entire boilerplates - each of which would be stored in its own repo on github. So if you do `yo assemble:boilerplate blog`, we can generate a new blog project using the blog boilerplate.

> it could be a good idea to allow for a verb-based documentation.

Awesome! At one point I had a question in generator-assemble that asked "Do you want to use verb for docs?" (see: https://github.com/jonschlinkert/generator-assemble/commit/2e751f11ad29061d5db016f9ebe60f54392140bb, only the hook is there now), but decided against it. I'm not opposed to reintroducing it, but... it might not be necessary b/c generator-verb can be used on existing projects. so you can run generator-assemble, then generator-verb, and get a more tailored result. I can go either way, it wouldn't take a lot to convince me lol!

**Comment by jonschlinkert** [comment]
btw, the toughest thing about verb so far has been creating "generic", reusable templates. I just quickly put a bunch of templates together and sorted them into these repos based on where it seemed to make sense:
- https://github.com/assemble/verb-contrib-templates (for the assemble core team, used with the `{%= contrib() %}` tag. I probably should have named this repo "verb-contrib-includes")
- https://github.com/assemble/verb-readme-includes (these are used with the `{%= include() %}` tag)

**Comment by Melindrea** [comment]
I think the big advantage having some kind of "we haz verb!" thing could bring is, well ... awareness. That might be as easy as just a "did you know you can build awesome docs using verb?" at the end.
